Failing test: Security Solution Cypress.x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/rule_details/backfill_group·cy·ts - Backfill groups should show backfill groups should show backfill groups

A test failed on a tracked branch

```
AssertionError: Timed out retrying after 150000ms: Expected to find element: `[data-test-subj="filters-global-container"] [data-test-subj="querySubmitButton"]`, but never found it.
    at refreshPage (webpack:///./tasks/security_header.ts:31:5)
    at cy.waitUntil.interval (webpack:///./tasks/create_new_rule.ts:886:18)
    at evaluate (webpack:///./support/commands.js:74:19)
    at Context.waitUntil (webpack:///./support/commands.js:83:9)
```

First failure: [kibana-on-merge - 8.19](https://buildkite.com/elastic/kibana-on-merge/builds/73731#01983d0e-667b-4cd0-9021-2f3d9bfd449d)

<!-- kibanaCiData = {"failed-test":{"test.class":"Security Solution Cypress.x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/rule_details/backfill_group·cy·ts","test.name":"Backfill groups should show backfill groups should show backfill groups","test.failCount":11}} -->

--- a/x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/rule_details/backfill_group.cy.ts
+++ b/x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/rule_details/backfill_group.cy.ts
@@ -31,8 +31,7 @@ import {
   FIRST_BACKFILL_ID,
 } from '../../../../tasks/api_calls/backfill';
 
-// Failing: See https://github.com/elastic/kibana/issues/229350
-describe.skip(
+describe(
   'Backfill groups',
   {
     tags: ['@ess', '@serverless', '@skipInServerlessMKI'],

